packages/schemas/tables/saml_application_secrets.sql

```diff
@@ -0,0 +1,22 @@
+/* init_order = 2 */
+
+create table saml_application_secrets (
+  id varchar(21) not null,
+  tenant_id varchar(21) not null
+    references tenants (id) on update cascade on delete cascade,
+  application_id varchar(21) not null
+    references applications (id) on update cascade on delete cascade,
+  private_key text not null,
+  certificate text not null,
+  created_at timestamptz not null default now(),
+  expires_at timestamptz not null,
+  active boolean not null,
+  primary key (id),
```

Comment:
(tenantId, applicationId, id) we need secrets to have a applicationId based index